docs/configuration.rst

```diff
@@ -29,27 +29,30 @@ PAPERLESS_REDIS=<url>
 
     Defaults to redis://localhost:6379.
 
+PAPERLESS_DBENGINE=<engine_name>
```

Comment:
Please add the options users can choose from (e.g. mariadb and postgresql).

Also, as is it right now, if this is empty, postgres will be used in the docker migration task. That should be documented.

Reply:
Added. Currently, it is expected that the engine will default to postgresql to maintain backwards compatibility and not interfere with existing/expected behavior.